hikari/impl/event_factory.py

```diff
@@ -571,13 +571,7 @@ def deserialize_message_reaction_remove_all_event(
                 message_id=message_id,
             )
 
-        # TODO: check if this can even occur.
-        return reaction_events.DMReactionDeleteAllEvent(
-            app=self._app,
-            shard=shard,
-            channel_id=channel_id,
-            message_id=message_id,
-        )
+        raise NotImplementedError("DM reaction all removes are not documented behavior")
```

Comment:
Yes it is?

Reply:
I haven't been able to trigger any of the two events 🤔

Reply:
They are documented behaviour though

Reply:
Hmmmm. We could just keep them ig